src/main/fc/config.h

```diff
@@ -72,6 +72,7 @@ typedef struct systemConfig_s {
     uint8_t activeRateProfile;
     uint8_t debug_mode;
     uint8_t task_statistics;
+    uint8_t cpu_overclock;
```

Comment:
uint8_t is consistent with other use. I've been toying with flags to save config space